src/components/RandomProductsBanner/RandomProductsBanner.js

```diff
@@ -1,9 +1,131 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createStructuredSelector } from 'reselect';
+import { FormattedMessage } from 'react-intl';
+
+import LoadingSpinner from '../LoadingSpinner';
+import messages from './messages';
+import { fetchBanner } from '../../containers/Dashboard/actions';
+import {
+  makeSelectProducts,
+  makeSelectCntProducts,
+  makeSelectLoading,
+  makeSelectError,
+} from '../../containers/Dashboard/selectors';
+import { randomProductsBannerOptions } from './constants';
+
+import './RandomProductsBanner.css';
+
+const { updateInterval, cntRandomProducts } = randomProductsBannerOptions;
+
+const getProductItem = product => {
+  return (
+    <li
+      key={product.get('id')}
+      className="random-products-banner__product-item"
+    >
+      <img
+        className="random-products-banner__product-item__img-preview"
+        src={product.get('img')}
+        alt={product.get('name')}
+        title={product.get('name')}
+      />
+      <div className="random-products-banner__product-item__info">
+        <h4 className="random-products-banner__product-item__name">
+          {product.get('name')}
+        </h4>
+        <span className="random-products-banner__product-item__type">
+          {product.get('type')}
+        </span>
+        <span className="random-products-banner__product-item__price">
+          <FormattedMessage {...messages.productPrice} />: $
+          {product.get('price')}
+        </span>
+        <span className="random-products-banner__product-item__description">
+          {product.get('description')}
+        </span>
+      </div>
+    </li>
+  );
+};
+
+const BannerView = data => {
+  const { products } = data;
+
+  return (
+    <React.Fragment>
+      <ul className="random-products-banner__products-list">
+        {products.map(product => {
+          return getProductItem(product);
+        })}
+      </ul>
+    </React.Fragment>
+  );
+};
 
 class RandomProductsBanner extends React.PureComponent {
+  componentDidMount() {
+    const { updateInterval } = this.props;
+    this.updateBanner();
+    this.interval = setInterval(this.updateBanner, updateInterval);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.interval);
+  }
+
+  updateBanner = () => {
+    const { cntRandomProducts, fetchBanner } = this.props;
+
+    fetchBanner(cntRandomProducts);
```

Comment:
`cntRandomProducts` is not a clear name. cnt is not clear to other developers. The code should be well read and clear to understand.